tests/jsmarionette/client/marionette-client/lib/marionette/client.js

```diff
@@ -468,20 +469,27 @@
         this._bypassScopeChecks = false;
       }
 
+      var data;
       if (this.protocol == 1) {
-        if (!cmd.to) {
-          cmd.to = this.actor || 'root';
+        if (!body.to) {
+          body.to = this.actor || 'root';
```

Comment:
Also could be written `body.to = body.to || this.actor || 'root';` but doesn't matter a ton :)